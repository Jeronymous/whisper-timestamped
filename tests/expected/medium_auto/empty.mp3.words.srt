1
00:00:00,020 --> 00:00:00,420
Thanks

2
00:00:00,420 --> 00:00:00,640
for

3
00:00:00,640 --> 00:00:02,980
watching!

