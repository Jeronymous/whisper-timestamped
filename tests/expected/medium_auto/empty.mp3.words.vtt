WEBVTT

00:00.020 --> 00:00.420
Thanks

00:00.420 --> 00:00.640
for

00:00.640 --> 00:02.980
watching!

