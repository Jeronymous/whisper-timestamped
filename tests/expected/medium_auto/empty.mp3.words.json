{
  "text": " Thanks for watching!",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.02,
      "end": 2.98,
      "text": " Thanks for watching!",
      "tokens": [
        50364,
        2561,
        337,
        1976,
        0,
        50514
      ],
      "temperature": 0.0,
      "avg_logprob": -0.899906975882394,
      "compression_ratio": 0.7142857142857143,
      "no_speech_prob": 0.6660081148147583,
      "words": [
        {
          "text": "Thanks",
          "start": 0.02,
          "end": 0.42
        },
        {
          "text": "for",
          "start": 0.42,
          "end": 0.64
        },
        {
          "text": "watching!",
          "start": 0.64,
          "end": 2.98
        }
      ]
    }
  ],
  "language": "en"
}